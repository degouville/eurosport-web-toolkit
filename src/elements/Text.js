import React from 'react'
import PropTypes from 'prop-types';

const Text = ({children, className}) => <span className={className}>{children}</span>;

Text.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Text;
