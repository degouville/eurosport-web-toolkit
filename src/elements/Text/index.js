import React from 'react';
import PropTypes from 'prop-types';

const Text = ({ children, ...props }) => <span {...props}>{children}</span>;

Text.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Text;
