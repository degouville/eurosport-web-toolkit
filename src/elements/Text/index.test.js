import React from 'react';
import { shallow } from 'enzyme';
import Text from '.';

it('renders a Text', () => {
  expect(shallow(<Text>A text</Text>)).toMatchSnapshot();
});
