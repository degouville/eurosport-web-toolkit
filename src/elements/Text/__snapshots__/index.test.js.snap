// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Text 1`] = `
<span>
  A text
</span>
`;
